appveyor.yml

```diff
@@ -67,6 +67,13 @@ before_test:
     - IF "%SULU_PHPCR_TRANSPORT%" == "jackrabbit" (
           php -dmemory_limit=-1 C:\tools\composer.phar require jackalope/jackalope-jackrabbit:~1.2 --no-update --no-interaction
       )
+    # for php < 7.4 we need to install package-versions as the new version
+    # of ocramius/package-versions does not support it and will end with
+    # the following error:
+    #
+    #     ocramius/package-versions only provides support for Composer 2 in 1.8+, which requires PHP 7.4
+    #     If you can not upgrade PHP you can require composer/package-versions-deprecated to resolve this with PHP 7.0+.
+    - php -dmemory_limit=-1 C:\tools\composer.phar require composer/package-versions-deprecated --no-update
```

Comment:
@sulu/core-team There is a [conflict](https://ci.appveyor.com/project/sulu/sulu/builds/33355141) with ocramius/package-versions. They had some discussion but ocramius does only support php 7.4 https://github.com/Ocramius/PackageVersions/pull/133.

So we basically some ways to avoid problems:

1. Require `composer/package-versions-deprecated` in sulu/sulu but this would avoid that newer version of `ocramius/package-versions` can be installed
2. Add `composer/package-versions-deprecated` as require dev in sulu and add `composer/package-versions-deprecated` to the skeleton. This would avoid that our tests run on `ocramius/package-versions` which I would like to avoid.
3. Add `composer/package-versions-deprecated` only in the CI where we need it and add `composer/package-versions-deprecated` to the `sulu/skeleton`
4. add `composer/package-versions-deprecated` only in the CI where we need it and add `composer/package-versions-deprecated` to the `sulu/skeleton` in a `post-root-package-install` for PHP versions lower then 7.4

Reply:
Seems like appveyor does currently install composer on 2.0, that because its downloaded from https://getcomposer.org/composer.phar thats why the error appears now but think we still should fix it as soon as possible before composer 2 is released. As the issues effects also all sulu versions using php < 7.4

Reply:
Will that be replaced by #5331?

Reply:
#5331 is a current workaround and we need to have a look at this before composer 2.0 is released.